frontend/src/Components/Player.jsx

```diff
@@ -38,12 +40,15 @@ export default function Player({ playerId, playerName, style, childRef }) {
                 socket.emit(`${gameState.role}-vote`, {
                     votingFor: playerName,
                 });
+                dispatch( { type:'show-selected' , status: `Selected ${playerName} for ability`, votedPlayer: playerName } );
                 break;
             case 'discussion':
                 socket.emit(`day-vote`, { votingFor: playerName });
+                dispatch( { type:'show-selected' , status: `Voted ${playerName} for trial`, votedPlayer: playerName } );
                 break;
             case 'trial':
                 socket.emit(`trial-vote`, { votingFor: playerName });
+                dispatch( { type:'show-selected' , status: `Voted to hang ${playerName}`, votedPlayer: playerName } );
```

Comment:
Could we please change 'hang' to 'kill'? :)

Reply:
done